lib/pf/Switch/Avaya.pm

```diff
@@ -589,8 +589,7 @@ sub radiusDisconnect {
         # merging additional attributes provided by caller to the standard attributes
         $attributes_ref = { %$attributes_ref, %$add_attributes_ref };
 
-        # Roles are configured and the user should have one
-        if (defined($role) && (defined($node_info->{'status'}) ) ) {
+        if ( $self->shouldUseCoA({role => $role}) ) {
```

Comment:
That is correct. It was likely to have been forgotten when RoleMap was brought in.
It should use the same logic as the other modules